databases/schemas/modlog-schema.sql

```diff
@@ -0,0 +1,26 @@
+ CREATE TABLE IF NOT EXISTS modlog (
+    modlog_id INTEGER NOT NULL PRIMARY KEY,
+    -- UNIX timestamp
+    timestamp INTEGER NOT NULL,
+    --- roomid OR global-roomid
+    roomid TEXT NOT NULL,
+    action TEXT NOT NULL,
+    visual_roomid TEXT,
+    action_taker TEXT,
```

Comment:
if this is a userid i'd suffix the column with that much like you did with `visual_roomid`